ui_tests/src/test/resources/com/saucedemo/account/logout.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Logout
+
+  @signIn
+  Scenario: Valid user sign in and logout
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
+    Then I am on Products page
+    When I click on menu icon on Header
+    And I click 'Logout' link on Menu
```

Comment:
@Arman0001 Merge of master branch into this branch should be performed in order to be able to use these steps